wire-ios/Wire-iOS/Sources/UserInterface/ConnectRequests/UserConnectionView.swift

```diff
@@ -114,36 +118,36 @@ final class UserConnectionView: UIView, Copyable {
          labelContainer,
-         firstLabel,
-         secondLabel].prepareForLayout()
+         guestIndicator,
+         guestWarningView].prepareForLayout()
 
         NSLayoutConstraint.activate([
             labelContainer.centerXAnchor.constraint(equalTo: centerXAnchor),
-            labelContainer.topAnchor.constraint(equalTo: topAnchor),
+            labelContainer.topAnchor.constraint(equalTo: topAnchor, constant: 16.0),
             labelContainer.leftAnchor.constraint(greaterThanOrEqualTo: leftAnchor),
 
-            userImageView.topAnchor.constraint(greaterThanOrEqualTo: labelContainer.bottomAnchor),
+            userImageView.topAnchor.constraint(equalTo: labelContainer.bottomAnchor, constant:  30.0),
             userImageView.centerXAnchor.constraint(equalTo: centerXAnchor),
-            userImageView.centerYAnchor.constraint(equalTo: centerYAnchor),
             userImageView.leftAnchor.constraint(greaterThanOrEqualTo: leftAnchor, constant: 54),
             userImageView.widthAnchor.constraint(equalTo: userImageView.heightAnchor),
-            userImageView.heightAnchor.constraint(lessThanOrEqualToConstant: 264),
-            userImageView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor),
+            userImageView.heightAnchor.constraint(equalToConstant: 200),
 
-            firstLabel.topAnchor.constraint(equalTo: labelContainer.topAnchor, constant: verticalMargin),
-            firstLabel.heightAnchor.constraint(equalToConstant: 16),
-            secondLabel.topAnchor.constraint(equalTo: firstLabel.bottomAnchor),
-            firstLabel.heightAnchor.constraint(equalToConstant: 16),
-            secondLabel.bottomAnchor.constraint(equalTo: labelContainer.bottomAnchor, constant: -verticalMargin),
+            guestIndicator.topAnchor.constraint(equalTo: userImageView.bottomAnchor, constant: 8.0),
+            guestIndicator.centerXAnchor.constraint(equalTo: centerXAnchor),
 
-            firstLabel.leadingAnchor.constraint(equalTo: labelContainer.leadingAnchor),
-            firstLabel.trailingAnchor.constraint(equalTo: labelContainer.trailingAnchor),
-
-            secondLabel.leadingAnchor.constraint(equalTo: labelContainer.leadingAnchor),
-            secondLabel.trailingAnchor.constraint(equalTo: labelContainer.trailingAnchor)
+            guestWarningView.topAnchor.constraint(equalTo: guestIndicator.bottomAnchor, constant: 30.0),
+            guestWarningView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 18.0),
+            guestWarningView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -23.0),
+            guestWarningView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor)
         ])
     }
+
+    private func updateGuestAccountViews() {
+        let viewer = SelfUser.current
+        let isGuest = !viewer.isTeamMember || viewer.canAccessCompanyInformation(of: user)
+        guestIndicator.isHidden = isGuest
+        guestWarningView.isHidden = isGuest && user.isConnected
```

Comment:
So we should show the guest warning view if the user is not a guest?

Reply:
i changed it for tests and forgot to set it back to correct code. fixed